src/index.js

```diff
@@ -45,7 +41,50 @@ server.on('after', (req, res, route, err) => {
   }
 });
 
-const startAPI = async () => {
+let qtumProcess;
+
+function startQtumProcess(reindex) {
+  // avoid using path.join for pkg to pack qtumd
+  const qtumdPath = `${path.dirname(__dirname)}/qtumd`;
+  const flags = ['-testnet', '-logevents', '-rpcuser=bodhi', '-rpcpassword=bodhi'];
+  if (reindex) {
+    flags.push('-reindex');
+  }
+
+  qtumProcess = spawn(qtumdPath, flags);
+  logger.debug(`qtumd started on PID ${qtumProcess.pid}`);
+
+  qtumProcess.stdout.on('data', (data) => {
+    logger.debug(`qtumd output: ${data}`);
+  });
+
+  qtumProcess.stderr.on('data', (data) => {
+    logger.error(`qtumd failed with error: ${data}`);
+
+    if (data.includes('You need to rebuild the database using -reindex-chainstate to enable -logevents.')) {
+      // Clean old process first
+      qtumProcess.kill();
+      qtumProcess = null;
+
+      // Restart qtumd with reindex flag
+      setTimeout(() => {
+        logger.info('Reindexing Qtum blockchain');
```

Comment:
`Restart and reindex Qtum blockchain`, probably need show this line in console

Reply:
changed to console.log